### PR TITLE
Linting, getting code rating A again

### PR DESCRIPTION
**Pull request by berland**
There are only a few accepted pylint exceptions added. 

* Allowing import to happen down in the code to avoid cycling import issues
* Allowing access to private functions in test code
* Allowing duplicate code in test code
* Allowing pointless statements and missing parameters in test code

### CODE REVIEW

**Comment by berland** [comment]
(looks like codacy is using a little tad old pylint, as it complains about import-outside-toplevel)

File: src/fmu/ensemble/realization.py [hunk 1]
```diff
@@ -1166,7 +1160,6 @@ def static_get_volumetric_rates(realization, column_keys, time_index, time_unit)
         This method is to be used by both ScratchRealization
         and VirtualRealization, and is documented there."""
         import calendar
```
Comment (asnyv) [inline src/fmu/ensemble/realization.py]: you don't need a `# pylint: disable=import-outside-toplevel` here?

File: src/fmu/ensemble/ensemble.py [hunk 2]
```diff
@@ -1587,17 +1580,16 @@ def _keyword_mean(self, prop, global_active, report=None):
...
-            mean.safe_div(global_active)
-            return mean
-        else:
             mean = EclKW(prop, len(global_active), EclDataType.ECL_FLOAT)
             for _, realization in six.iteritems(self._realizations):
-                mean += realization.get_global_init_keyword(prop)
+                mean += realization.get_global_unrst_keyword(prop, report)
             mean.safe_div(global_active)
             return mean
+        mean = EclKW(prop, len(global_active), EclDataType.ECL_FLOAT)
```
[the remark this answers is not in the document]
Comment (berland) [reply src/fmu/ensemble/ensemble.py]: Good catch. Fixed

(this is "legacy" code for fmu.ensemble, inherited from its predecessor, and is very weakly tested)

File: src/fmu/ensemble/ensemble.py [hunk 3]
```diff
@@ -1615,14 +1607,12 @@ def _keyword_std_dev(self, prop, global_active, mean, report=0):
...
             return std_dev.isqrt()
-
-        else:
-            std_dev = EclKW(prop, len(global_active), EclDataType.ECL_FLOAT)
-            for real, realization in six.iteritems(self._realizations):
-                real_prop = realization.get_global_init_keyword(prop)
-                std_dev.add_squared(real_prop - mean)
-            std_dev.safe_div(global_active)
-            return std_dev.isqrt()
+        std_dev = EclKW(prop, len(global_active), EclDataType.ECL_FLOAT)
```
[the remark this answers is not in the document]
Comment (berland) [reply src/fmu/ensemble/ensemble.py]: Fixed, as above.

File: src/fmu/ensemble/realization.py (hunk 1 above)
Comment (berland) [reply src/fmu/ensemble/realization.py]: Moved to top of file, no circular import problems here.

**Pull request merged**